docs/docs-ref-autogen/excel/excelscript/excelscript.sortdataoption.yml

```
@@ -4,7 +4,35 @@ uid: 'ExcelScript!ExcelScript.SortDataOption:enum'
 package: ExcelScript!
 fullName: ExcelScript.SortDataOption
 summary: ''
-remarks: ''
+remarks: |-
+
+
+  #### Examples
+
+  ```TypeScript
+  /**
+   * This script sorts a table based on the values in column 1.
+   * If the text of a column-1 value can be treated as a number, 
+   * it will be sorted in numerical order, rather than alphabetical 
```

This phrase throws me off because "alphabetical" sorting is being applied to a number. Would "alphanumeric" be appropriate? Or/in addition, I recommend an expanded definition of the numerical and alphabetical sort types (unless a typical Office Scripts customer knows these terms). 

```suggestion
   * it will be sorted in numerical order, rather than alphanumerical 
```

---

Thanks Alison!

I used "alphabetical" because the default sorting operates based on the unicode value of characters, whereas this option lets you specify that the text can be numbers. I'll change "alphabetical" to "Unicode order". That said, I think the preceding context and following "123 > 12.3" example provide more clarity than I could expanding the terms' definitions. Especially because the scenario here is that someone has set the NumberFormatCategory to "Text" and still wants to compare values as numbers. 

Side-note, this SortDataOption enum has no description in either Excel or ExcelScript. I only figured out what this did through experimentation.